src/service.rs

```diff
@@ -62,11 +63,31 @@ pub(crate) enum InnerProtocolHandle {
     Session(Box<dyn SessionProtocol + Send + 'static>),
 }
 
+struct SessionController {
+    inner: Arc<SessionContext>,
+    event_sender: mpsc::Sender<SessionEvent>,
+    quick_sender: mpsc::Sender<SessionEvent>,
+}
+
+impl SessionController {
+    fn try_send(
```

Comment:
I think this function is not necessary, just get the sender by priority is ok.

Reply:
This change can avoid fields(`quick_sender`, `event_sender`) being exposed outside. So that keeps the API clean.